cezo_fl_main.py

```diff
@@ -70,6 +71,7 @@ def prepare_settings_underseed(args, device):
         large_model = args.large_model
         model_name = SUPPORTED_LLM[large_model]
         model = AutoModelForCausalLM.from_pretrained(model_name, torch_dtype=torch_dtype).to(device)
+        replace_penultimate_layer(model)
```

Comment:
please do not modify the cezo_fl main file. It should be place in a separate file.

Reply:
I will create a new independent file for this.